src/pretix/api/views/event.py

```diff
@@ -217,9 +217,10 @@ def is_future_qs(self, queryset, name, value):
             return queryset.exclude(expr)
 
 
-class SubEventViewSet(ConditionalListView, viewsets.ReadOnlyModelViewSet):
+class SubEventViewSet(ConditionalListView, viewsets.ModelViewSet):
     serializer_class = SubEventSerializer
     queryset = ItemCategory.objects.none()
+    permission_classes = (EventCRUDPermission,)
```

Comment:
You don't need permission_classes (this is special for events as their creation works differently), you can just do:
```
    write_permission = 'can_change_event_settings'
```